operations/normalize_nodes.yml

```diff
@@ -0,0 +1,15 @@
+id: normalize_nodes
```

Comment:
Let's make this a non-unique operation (so there is only one way to do the normalization)